fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatPermissionEvaluator.java

```diff
@@ -188,19 +189,18 @@ public boolean canCreate(String resourceType, Object resource) {
                       try {
                         fiatService.canCreate(username, resourceType, resource);
                         return true;
-                      } catch (RetrofitError re) {
+                      } catch (NotFoundException e) {
+                        return false;
+                      } catch (SpinnakerHttpException e) {
                         boolean shouldRetry = true;
-                        if (re.getKind() == RetrofitError.Kind.HTTP) {
-                          switch (HttpStatus.valueOf(re.getResponse().getStatus())) {
-                            case NOT_FOUND:
-                              return false;
-                            case BAD_REQUEST:
-                              shouldRetry = false;
-                          }
+                        if (e.getResponseCode() == HttpStatus.BAD_REQUEST.value()) {
+                          shouldRetry = false;
                         }
-                        IntegrationException ie = new IntegrationException(re);
-                        ie.setRetryable(shouldRetry);
-                        throw ie;
+                        e.setRetryable(shouldRetry);
+                        throw e;
+                      } catch (SpinnakerServerException e) {
```

Comment:
Plus also, [SpinnakerServerException](https://github.com/spinnaker/kork/blob/b6fa1103b7b3b12734af39cb6f6ca2b04cdd59d6/kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/exceptions/SpinnakerServerException.java) doesn't mention retryable, so from [SpinnakerException](https://github.com/spinnaker/kork/blob/b6fa1103b7b3b12734af39cb6f6ca2b04cdd59d6/kork-exceptions/src/main/java/com/netflix/spinnaker/kork/exceptions/SpinnakerException.java), retryable is null...and then RetrySupport [retries](https://github.com/spinnaker/kork/blob/b6fa1103b7b3b12734af39cb6f6ca2b04cdd59d6/kork-exceptions/src/main/java/com/netflix/spinnaker/kork/core/RetrySupport.java#L38-L40).